Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -16,6 +16,8 @@
 
 # disable insecure warnings
 urllib3.disable_warnings()
+v21_logger = logging.getLogger("taxii2client.v21")
+v21_logger.setLevel(logging.CRITICAL)
```

Comment:
as discussed let's check if this can be passed to demisto.debug instead. if not, leave as is.